website/docs/reference/snapshot-properties.md

```diff
@@ -41,3 +41,37 @@ snapshots:
 
 ```
 </File>
+
+
+## Define snapshots in YAML
+
+Defining a snapshot in YAML treats snapshots as a distinct resource type, separate from models, allowing for improved organization and consistency within your code.
```

Comment:
This makes it sound like defining snapshots in YML is _what_ treats snapshots as a distinct resource type, which is not true.